<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>440</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu,sans-serif</family>
      <pointsize>-1</pointsize>
      <weight>87</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">    font: 100%/1.4 Lato, sans-serif;
    color: #333;
    box-sizing: inherit;
    font-family: Ubuntu, sans-serif;
    font-weight: 700;
    line-height: 1;
    font-size: 26px;
    margin: .7em 0 .2em;
    clear: both;</string>
    </property>
    <property name="text">
     <string>CADASTRAR COMPRAS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>640</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 0, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>CADASTRAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>390</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cartão de crédito/ débito:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>630</y>
      <width>60</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 0, 255);
background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>SAIR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Parcelas:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ANO:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Ano">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>179</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: qlineargradient(spread:pad, x1:0.504, y1:0, x2:0.504, y2:0.981, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));
    color: rgb(255, 255, 255);
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    border-color: rgb(0, 0, 0);
}

QPushButton:hover {
    background-color: qlineargradient(spread:pad, x1:0.504, y1:0, x2:0.504, y2:0.981, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(200, 0, 0, 255));
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0.504, y1:0, x2:0.504, y2:0.981, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(150, 0, 0, 255));
}
</string>
    </property>
    <property name="text">
     <string>+1000</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Categoria:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>140</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>179</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: qlineargradient(spread:pad, x1:0.504, y1:0, x2:0.504, y2:0.981, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));
    color: rgb(255, 255, 255);
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    border-color: rgb(0, 0, 0);
}

QPushButton:hover {
    background-color: qlineargradient(spread:pad, x1:0.504, y1:0, x2:0.504, y2:0.981, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(200, 0, 0, 255));
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0.504, y1:0, x2:0.504, y2:0.981, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(150, 0, 0, 255));
}
</string>
    </property>
    <property name="text">
     <string>+1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Valor:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <item>
     <property name="text">
      <string>Outros</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Alimentação</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Moradia</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Transporte</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Saúde</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Educação</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Lazer</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vestuário</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Comunicação</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Impostos</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dívidas</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>179</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: qlineargradient(spread:pad, x1:0.504, y1:0, x2:0.504, y2:0.981, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));
    color: rgb(255, 255, 255);
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    border-color: rgb(0, 0, 0);
}

QPushButton:hover {
    background-color: qlineargradient(spread:pad, x1:0.504, y1:0, x2:0.504, y2:0.981, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(200, 0, 0, 255));
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0.504, y1:0, x2:0.504, y2:0.981, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(150, 0, 0, 255));
}
</string>
    </property>
    <property name="text">
     <string>+100</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Data:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>179</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: qlineargradient(spread:pad, x1:0.504, y1:0, x2:0.504, y2:0.981, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));
    color: rgb(255, 255, 255);
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    border-color: rgb(0, 0, 0);
}

QPushButton:hover {
    background-color: qlineargradient(spread:pad, x1:0.504, y1:0, x2:0.504, y2:0.981, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(200, 0, 0, 255));
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0.504, y1:0, x2:0.504, y2:0.981, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(150, 0, 0, 255));
}
</string>
    </property>
    <property name="text">
     <string>+10</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Descrição:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: qlineargradient(spread:pad, x1:0.504, y1:0, x2:0.504, y2:0.981, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));
    color: rgb(255, 255, 255);
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    border-color: rgb(0, 0, 0);
}

QPushButton:hover {
    background-color: qlineargradient(spread:pad, x1:0.504, y1:0, x2:0.504, y2:0.981, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(200, 0, 0, 255));
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0.504, y1:0, x2:0.504, y2:0.981, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(150, 0, 0, 255));
}
</string>
    </property>
    <property name="text">
     <string>+1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: qlineargradient(spread:pad, x1:0.504, y1:0, x2:0.504, y2:0.981, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));
    color: rgb(255, 255, 255);
    border-radius: 15px;
    border-style: solid;
    border-width: 2px;
    border-color: rgb(0, 0, 0);
}

QPushButton:hover {
    background-color: qlineargradient(spread:pad, x1:0.504, y1:0, x2:0.504, y2:0.981, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(200, 0, 0, 255));
}

QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0.504, y1:0, x2:0.504, y2:0.981, stop:0 rgba(85, 0, 255, 255), stop:1 rgba(150, 0, 0, 255));
}
</string>
    </property>
    <property name="text">
     <string>+10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>580</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 0, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>GERENCIAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>547</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
